hop/__main__.py

```diff
@@ -8,22 +8,51 @@
 from . import version
 
 
-def append_subparser(subparser, cmd, func):
+def append_subparser(subparser, cmd, func, formatter=False):
```

Comment:
Maybe we can change formatter parameter to apply_formate?

Reply:
I modified this in the latest batch of commits.